src/referentielijsten/api/tests/test_tabellen_admin.py

```diff
@@ -0,0 +1,48 @@
+from django.contrib.admin.sites import AdminSite
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from django_webtest import WebTest
+from maykin_2fa.test import disable_admin_mfa
+
+from referentielijsten.accounts.tests.factories import UserFactory
+
+from ..admin import TabelAdmin
+from ..models import Tabel
+from .factories import TabelFactory
+
+
+@disable_admin_mfa()
+class TabellenAdminTests(WebTest):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        cls.superuser = UserFactory.create(superuser=True)
+
+    def test_items_link_returns_correct_admin_url(self):
+        site = AdminSite()
+        admin = TabelAdmin(Tabel, site)
+        tabel = TabelFactory.create()
+
+        link = admin.items_link(tabel)
+
+        # TODO replace with reverse(..., query=...) once Django is upgraded to 5.2
+        expected_url = reverse("admin:api_item_changelist") + f"?tabel__id={tabel.id}"
+        expected_html = (
+            f'<a href="{expected_url}">{_("Show items in item list view")}</a>'
+        )
+
+        self.assertHTMLEqual(link, expected_html)
+
```

Comment:
Instead of manually setting up the admin, you can use webtest here to simulate what a user would do by making a request to the tabel changelist (not 100% sure if the assertion I do here is correct, but something like this):

```suggestion
    # TODO replace with reverse(..., query=...) once Django is upgraded to 5.2
    expected_url = reverse("admin:api_item_changelist") + f"?tabel__id={tabel.id}"
    expected_html = (
        f'<a href="{expected_url}">{_("Show items in item list view")}</a>'
    )
    
    response = self.app.get(
        reverse("admin:api_tabel_changelist", user=self.superuser)
    )

    self.assertIn(expected_html, response.content)
```